python/pyspark/rdd.py

```diff
@@ -2067,7 +2067,7 @@ def add_shuffle_key(split, iterator):
                     avg = int(size / n) >> 20
                     # let 1M < avg < 10M
                     if avg < 1:
-                        batch *= 1.5
+                        batch = min(sys.maxsize, batch * 1.5)
```

Comment:
should we use `sys.float_info.max` instead, @nolanliou? Hm, btw just realised that it's funny that it can reach to the maximum of float ...

Reply:
Anyway avoiding failure on the batch size makes sense.

Reply:
I think `sys.maxsize`  is ok. 

It’s not easy to encounter this problem, but I ran into it...

Reply:
okay, `sys.maxsize` batch size already doesn't make much sense anyway.